### PR TITLE
DEIMOS frame-typing tweaks and citation policy

### PR DESCRIPTION
**Pull request by kbwestfall**
Minor PR that includes:

- Frame-typing tweaks for DEIMOS
    - Exposure-time ranges removed
    - All frame types now key off OBSTYPE
- Added more detail on citation policy to main page on readthedocs

Tests pass.  I'm not planning to run the devsuite, but can if prompted.

```
======================= 246 passed in 1083.89s (0:18:03) =======================
```

### CODE REVIEW

**Review by profxj — APPROVED** [review]
Thanks for this.

**Comment by kbwestfall** [comment]
Now has much more in it given the merge of #1035 .  Tests still pass:

```
======================= 246 passed in 855.57s (0:14:15) ========================
```
 I'm not planning to run the devsuite, but can if prompted.

File: pypeit/scripts/chk_edges.py [hunk 1]
```diff
@@ -22,15 +23,12 @@ def parser(options=None):
     parser.add_argument('--try_old', default=False, action='store_true',
                         help='Attempt to load old datamodel versions.  A crash may ensue..')
 
+    if return_parser:
+        return parser
+
     return parser.parse_args() if options is None else parser.parse_args(options)
```
Comment (jhennawi) [inline pypeit/scripts/chk_edges.py:30]: I feel this TODO should remain. Either we need a separate check edges for the slit_trace object script, or the show function should be moved there.

**Review by jhennawi — CHANGES_REQUESTED** [review]
Thanks for the work here @kbwestfall. Very minor changes requested.

File: pypeit/core/wavecal/wv_fitting.py [hunk 2]
```diff
@@ -339,7 +339,7 @@ def iterative_fitting(spec, tcent, ifit, IDs, llist, disp,
     # Final fit (originals can now be rejected)
```
[the remark this answers is not in the document]
Comment (rcooke-ast) [reply pypeit/core/wavecal/wv_fitting.py:339]: I believe this has been pushed to master, but has not been pushed to develop. I've included it in PR #1027, but perhaps it's best that it goes in with the next merged PR.

[the remark this answers is not in the document]
Comment (kbwestfall) [reply pypeit/core/wavecal/wv_fitting.py:339]: Yeah, the PR also effectively updates `develop` with changes made to `master`.

File: pypeit/scripts/chk_edges.py (hunk 1 above)
Comment (kbwestfall) [reply pypeit/scripts/chk_edges.py:30]: Added back.

**Comment by kbwestfall** [comment]
Thanks for having a look, @jhennawi .  I've addressed your comments.

**Review by jhennawi — APPROVED** [review]

**Pull request merged**